share/completions/krita.fish

```diff
@@ -0,0 +1,56 @@
+function __krita_append_image_format_value -a value current_token
+    echo $value |
```

Comment:
`string` can take the value on the commandline, so the `echo` isn't useful:

`string replace -r '^' -- $current_token $value`